doc/api/vm.md

```diff
@@ -1573,6 +1573,105 @@ inside a `vm.Context`, functions passed to them will be added to global queues,
 which are shared by all contexts. Therefore, callbacks passed to those functions
 are not controllable through the timeout either.
 
+## Local Worker
+
+> Stability: 1 - Experimental
+
+### Class: `LocalWorker`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Extends: {EventEmitter}
+
+A `LocalWorker` is effectively a Node.js environment that runs within the
+same thread.
```

Comment:
I need a bit more detail than this. A Node.js environment . . . with its own global scope? That can have separate `NODE_OPTIONS`? Is it CommonJS or ESM, or either?

To others’ points, how does this differ from Realm?

Reply:
A Realm (in specs terms) does not support ESM, `require` and all Node.js core modules. It's like a Realm, but with all the Node.js stuff.